styles.go

```diff
@@ -2872,6 +2872,32 @@ func (f *File) SetConditionalFormat(sheet, area, formatSet string) error {
 	return err
 }
 
+// GetConditionalFormat() retrieves all comments and returns a map of worksheet name to
+// the worksheet conditioanl formats in xml string.
+func (f *File) GetConditionalFormat() (map[string][]string, error) {
+	resultCF := map[string][]string{}
+	for sheet := range f.sheetMap {
+		ws, err := f.workSheetReader(sheet)
+		if err != nil {
+			return resultCF, err
+		}
+
+		resultCF[sheet] = []string{}
+		for _, cf := range ws.ConditionalFormatting {
+			cfRule := CfRule{SQRef: cf.SQRef}
+			for _, cr := range cf.CfRule {
+				cfRule.CfRule = *cr
+				cfRuleBytes, err := xml.Marshal(cfRule)
```

Comment:
The result of this function should be same with the parameter `formatSet` of the `SetConditionalFormat` function. In another word, that the serilized string of `formatConditional`. So I think the new exported data type `CfRule` is not required.